DBs/mariaDB4j-db-11.4.5/mariaDB4j-db-winx64-11.4.5/pom.xml

```
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>ch.vorburger.mariaDB4j</groupId>
+    <artifactId>mariaDB4j-pom-lite</artifactId>
+    <version>2.2.2</version>
+    <relativePath>../../../mariaDB4j-pom-lite/pom.xml</relativePath>
+  </parent>
+
+  <artifactId>mariaDB4j-db-winx64</artifactId>
+  <version>11.4.5</version>
+
+  <properties>
+    <mariaDB.version>11.4.5</mariaDB.version>
+    <cacheDir>${project.basedir}/../../../tmpDir</cacheDir>
+  </properties>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>${project.build.directory}/generated-resources</directory>
+      </resource>
+    </resources>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.8</version>
```

```suggestion
        <version>3.1.0</version>
```

@TheKnowles I bumped this in the Linux (and now macOS as well!) ones - do you want to do that here as well?

---

Agree. Old version from copy paste from 10.6.x winx pom.